metrics/scaling/k8s_scale.sh

```diff
@@ -39,68 +39,68 @@ TEST_NAME="k8s scaling"
 grab_stats() {
 	local launch_time_ms=$1
 	local n_pods=$2
-    local cpu_idle=()
-    local mem_free=()
+	local cpu_idle=()
+	local mem_free=()
 	info "And grab some stats"
 
-    local pods_json="$(cat << EOF
-            "n_pods": {
-                "Result": ${n_pods},
-                "Units" : "int"
-            }
+	local pods_json="$(cat << EOF
+			"n_pods": {
```

Comment:
heh - it is really hard to try and locally format the json txt in the shell files so they generate pretty json in the final output ;-).
hmm, maybe we could spit the json out via a json pretty printer stream?